sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryIO.java

```diff
@@ -2138,6 +2153,11 @@ public boolean getValidate() {
         return validate;
       }
 
+      /** Returns {@code true} if ignoreUnknownValues is enabled. */
+      public Boolean getIgnoreUnknownValues() {
```

Comment:
Nit: Mark the return value as `@Nullable`.

Throughout, either `@Nullable Boolean` or, if it cannot be null, `boolean`.